assets/python/matplotlib_basic.pyi

```diff
@@ -0,0 +1,90 @@
+# -------------------------------------------------------------------------------
```

Comment:
Regarding the naming, we adopted a more explanatory convention in https://exabyte.atlassian.net/wiki/spaces/PD/pages/1791263043/ML21+Workflow+Building+Blocks. See `pyml:pre_processing:standardization:sklearn`, for instance.

Maybe: `python:post_processing:plot:matplotlib`?

Reply:
The file name should be adjusted too, same as in https://github.com/Exabyte-io/exabyte-application-flavors/tree/epic/SOF-6265/assets/python/ml.